### PR TITLE
Deploy magnum/queens on openstack/pike

### PR DESCRIPTION
**Pull request by brtkwr**
Deployment instructions:

Add public key for  `$USER` into `authorized_keys` for `stack` user on all hosts (`10.41.253.[100: seed | 101: prod controller | 102: prod monitor | 103: alt-1 controller]`):

```
kayobe overcloud container image build magnum --push
```

Before executing the next command:
comment out `kolla_openstack_release: 6.0.0.0` on `kolla.yml`
add public key for `$USER` also into `kolla` user on `alt-1 controller`

```
kayobe overcloud container image pull --kolla-tags magnum
kayobe overcloud service upgrade --kolla-tags magnum 
```

### CODE REVIEW

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -12,10 +12,12 @@
 
 # URL of Kolla source code repository if type is 'source'.
 #kolla_source_url:
+kolla_source_url: https://git.openstack.org/openstack/kolla
 
 # Version (branch, tag, etc.) of Kolla source code repository if type is
 # 'source'.
 #kolla_source_version:
+kolla_source_version: stable/queens
```
Comment (dougszumski) [inline etc/kayobe/kolla.yml:20]: Are we upgrading alt-1 as a whole to queens, or is this only for building Magnum from queens and using that with the existing deployment?

Comment (brtkwr) [reply etc/kayobe/kolla.yml:20]: Only for building Magnum from queens AFAIK

Comment (dougszumski) [reply etc/kayobe/kolla.yml:20]: I don't think we want to make this change in that case as if someone builds an image for another service, that'll also be queens. You should be able to temporarily override it, push the queens magnum image, and leave it unchanged here?

Comment (markgoddard) [reply etc/kayobe/kolla.yml:20]: +1

File: etc/kayobe/kolla.yml [hunk 2]
```diff
@@ -71,6 +73,7 @@
 
 # Kolla OpenStack release version. This should be a Docker image tag.
 #kolla_openstack_release:
+kolla_openstack_release: 6.0.0.0
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml:76]: This one also needs to go.

File: etc/kayobe/kolla/passwords.yml [hunk 3]
```diff
@@ -1,1010 +1,1010 @@
 $ANSIBLE_VAULT;1.1;AES256
```
Comment (markgoddard) [inline etc/kayobe/kolla/passwords.yml:1]: Do you know what has changed in this file? You can use ansible-vault decrypt and diff to check.

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: etc/kayobe/kolla.yml (hunk 1 above)
Comment (brtkwr) [reply etc/kayobe/kolla.yml:20]: Do I need to specify kolla_source_version to run the following?
```
	kayobe overcloud container image pull --kolla-tags magnum
	kayobe overcloud service upgrade --kolla-tags magnum 
```

Comment (dougszumski) [reply etc/kayobe/kolla.yml:20]: I don't think so. By that stage the image will have been built and pushed so it *shouldn't* matter that kolla is on pike.

**Comment by brtkwr** [comment]
Apologies, I intended to rename the branch but didn't realise that would close this pull request. New pull request is here: https://github.com/SKA-ScienceDataProcessor/alaska-kayobe-config/pull/61/files

**Pull request closed without merging**